bot/code_review_bot/workflow.py

```diff
@@ -197,6 +219,11 @@ def publish(self, revision, issues, task_failures, notices, reviewers):
         # Only publish errors and "in patch" warnings due to a backend timeout
         publishable_issues = [i for i in issues if i.is_publishable()]
 
+        if revision.before_after_feature:
+            # In case before/after feature is enabled, only store issues that are new
+            # That is required to specify the number of unresolved or closed issues
+            publishable_issues = [i for i in publishable_issues if i.new_issue]
```

Comment:
Currently we show in-patch issues even if they are not new, maybe we should keep that behavior (so developers could fix them as ride-along to their changes).